src/conversion/viewconversiondispatcher.js

```diff
@@ -180,6 +179,30 @@ export default class ViewConversionDispatcher {
 			this.fire( 'documentFragment', data, consumable, this.conversionApi );
 		}
 
+		// Handle incorrect `data.output`.
+		if ( data.output === undefined ) {
```

Comment:
It seemed correct when I was working just on the dispatcher but "in real life" I can see that this is actually incorrect. There are situations where we expect that the view element will not be converted: for example when text is wrong with schema. On the other hand we have "default" converters which will take care of "non-intentionally not-converted elements". So yeah, this should be removed.